syntax = "proto3";

package sentinel_ai;

message RankedEvent {
  string id = 1;
  string title = 2;
  string timestamp = 3;
  string source = 4;
  repeated string categories = 5;
  bool is_relevant = 6;
  float importance_score = 7;
  float recency_score = 8;
  float final_score = 9;
}
